README.md

```diff
@@ -99,6 +99,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [threeschwifty](https://ngokevin.github.com/threeschwifty) &mdash; [Code](https://github.com/ngokevin/threeschwifty) - 360-degree photo and video viewer.
 - [VR GIF](http://x627.com/vr-gif/) &mdash; [Post](http://x627.com/a-frame-is-awesome/) - GIFs in VR.
 - [Virtual Art School Tour](http://stampswebdesign.com/withersc/hud/index.html) (by [@withersc]) - 360-degree photos with navigation.
+- [360-degree photosphere viewer example](https://github.com/IdeaSpaceVR/aframe-360-degree-photosphere)
```

Comment:
We started to follow the general guidelines of sindresorhus/awesome. Can you check out the [contributing guidelines](https://github.com/aframevr/awesome-aframe/blob/master/contributing.md#adding-to-this-list) and modify things such as Title Casing and Description. Thanks!